<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YHT Ticket Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>     Kontrol Bilgileri     </string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Chrome Driver :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="chromeDriverLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Kalkış İstasyonu: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="departureStationCombo">
        <item>
         <property name="text">
          <string>Ankara Gar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Eryaman</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polatlı</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Eskişehir</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bozüyük</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bilecik</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Arifiye</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>İzmit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gebze</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pendik</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bostancı</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Söğütlüçeşme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bakırköy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Halkalı</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Varış İstasyonu :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="arrivalStationCombo">
        <item>
         <property name="text">
          <string>Halkalı</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bakırköy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Söğütlüçeşme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bostancı</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pendik</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gebze</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>İzmit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Arifiye</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bilecik</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bozüyük</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Eskişehir</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polatlı</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Eryaman</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ankara Gar</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tarih :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Kontrol Aralığı (Dk.) : </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="controlTimeSpinBox">
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="departureStationDateTimeEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="addChromeDriverButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>28</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ChromeDriver</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startControlButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Başlat</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>     Kontrol Geçmişi     </string>
    </property>
    <widget class="QPlainTextEdit" name="logPlainTextEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
